Meta/CMake/vcpkg/official-triplets/x64-linux.cmake

```diff
@@ -0,0 +1,2 @@
+include (${CMAKE_CURRENT_LIST_DIR}/../base-triplets/x64-linux.cmake)
```

Comment:
If we want to not confuse upstream vcpkg bug report viewers, we should suffix our dynamic triplets with -dynamic and have this one be the "standard". Dynamic lib triplets are considered "community triplets" by the vcpkg team and CI

Reply:
No real preference on what the presets are named tho, those are arbitrary

Reply:
Sounds like a separate change then? If this one stays "x64-linux", we can rename the other triplets to be "x64-linux-dynamic" etc. separately